src/fosslight_util/oss_item.py

```
@@ -177,15 +177,12 @@ def get_print_json(self):
-    except Exception:
-        try:
-            f = open(source_name_or_path, "rb")
-            byte = f.read()
-            checksum = str(hashlib.sha1(byte).hexdigest())
-            f.close()
-        except Exception as ex:
-            _logger.info(f"(Error) Get_checksum: {ex}")
+        if os.path.isfile(source_name_or_path):
+            with open(source_name_or_path, "rb") as f:
+                byte = f.read()
+                checksum = str(hashlib.sha1(byte).hexdigest())
```

Ruff (0.15.20)

[error] Probable use of insecure hash functions in `hashlib`: `sha1`

(S324)